Mathlib/FieldTheory/Finiteness.lean

```diff
@@ -84,6 +84,14 @@ theorem range_finsetBasis [IsNoetherian K V] :
 
 variable {K V}
 
+-- should this go in a namespace?
```

Comment:
Good question... I think `FiniteDimensional` is a good idea even if it doesn't appear in the statement.

Reply:
I'd vote for `Module.`, `Fintype.` would be the next.

Reply:
`Module` looks a good idea. Thanks!

bors d+